tests/integration/servicemesh/smmr/smmr.go

```diff
@@ -0,0 +1,258 @@
+//go:build integ
+// +build integ
+
+//
+// Copyright Red Hat, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package servicemesh
```

Comment:
shouldn't this be `smmr`?

Reply:
I think that smmr.go and smmt_test.go should belong to package named `smmr`, but federation test belongs to `servicemesh` package, so I implemented smmr test following the existing convention.

Reply:
https://github.com/maistra/istio/blob/maistra-2.1/tests/integration/servicemesh/federation/federation.go#L16

Reply:
oh, I see- that was a mistake on my part

Reply:
fixed